councilmatic_core/management/commands/import_data.py

```diff
@@ -3103,11 +3098,15 @@ def executeTransactionList(self, query_list, args_list):
             for query, args in zip(query_list, args_list):
                 self.connection.execute(query, *args)
 
-    # OCD API has intermittently thrown 502 errors; only proceed when receiving an 'ok' status
-    def _get_response(self, url):
-        response = session.get(url)
+    # OCD API has intermittently thrown 502 and 504 errors; only proceed when receiving an 'ok' status.
+    def _get_response(self, url, params=None, timeout=60, error=True, **kwargs):
```

Comment:
Might I suggest something like `raise_error` rather than `error` as the keyword argument here? Seems a bit more descriptive.

Reply:
Oh, good!